### PR TITLE
Bug 1136988 - Catch system resize event whenever rocketbar is active

### PR DESCRIPTION
**Pull request by alivedise**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[alivedise Alive.Kuo (alivedise)](https://github.com/alivedise) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8993346b103e)

File: apps/system/test/unit/rocketbar_test.js [hunk 1]
```diff
@@ -666,6 +666,13 @@ suite('system/Rocketbar', function() {
     test('Search window without front window', function() {
       var respond =
         subject.respondToHierarchyEvent(new CustomEvent('system-resize'));
+      assert.isFalse(respond);
+    });
+
+    test('Rocketbar is inactive', function() {
+      subject.deactivate();
+      var respond =
```
Comment (etiennesegonzac) [inline apps/system/test/unit/rocketbar_test.js]: nit: for all those tests let's rename `respond` to `stopsPropagation`, it makes the tests more readable / document what we're actually asserting.

Comment (alivedise) [inline apps/system/test/unit/rocketbar_test.js]: done

**Comment by try-server-hook** [comment]
[alivedise Alive.Kuo (alivedise)](https://github.com/alivedise) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2b993625a00c)

**Pull request merged**